src/Hal/Bundle/PhpMetricsCollector/DependencyInjection/Configuration.php

```diff
@@ -4,11 +4,6 @@
 use Symfony\Component\Config\Definition\Builder\TreeBuilder;
 use Symfony\Component\Config\Definition\ConfigurationInterface;
 
-/**
- * This is the class that validates and merges configuration from your app/config files
- *
- * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html#cookbook-bundles-extension-config-class}
- */
 class Configuration implements ConfigurationInterface
 {
     /**
```

Comment:
And below, in the getConfigTreeBuilder() method, you need to replace its content by this:
`        $treeBuilder = new TreeBuilder('phpmetrics');
        return $treeBuilder;`